.gitignore

```diff
@@ -13,7 +13,7 @@ web/css/
 web/media
 web/assetic
 build/
-vendor/
+/vendor/
```

Comment:
oh well. i recently read quite some tweets of people saying to use npm instead. i am rather unhappy with the amount of vendor code we need to commit here. is there a compelling reason to not use cdn versions of that stuff? without a grunt/gulp pipeline to combine assets, we don't win much by serving the files locally anyways. and i would want to avoid taking frontend tool decisions in the sandbox if we can.